### DISCUSSION TITLE
how can I show my email in my github?

### DISCUSSION THREAD

**Discussion by githubbergirl** [community/community#36793] [Other Feature Feedback, Questions, & Ideas] [answered]
hey guys, 
I'm having problem with showing my email in my bio I have seen people do that but I'm not able to show it.
I have also verified my email!

**Comment by matajeu** [comment]
Well... Please check this.
Setting -> Emails -> Keep my email addresses private (uncheck)
and then, try setting up your email again on the profile page

**Comment by kude5n1k** [comment]
Hi friend!
1. Profile page -> Setting(upper right corner) -> Emails -> Keep my email addresses private (uncheck)
2. Profile page -> Edit profile(below your avatar) -> write your email in a special field -> Save

**Comment by shirinapr** [comment, marked as answer]
There is a similar discussions[ here](https://github.com/orgs/community/discussions/36705).

solution was:
1) verify your email
2)  go to [settings](https://github.com/settings/emails) read under and uncheck "Keep my email addresses private".
3)   add your github email in your git config.

*  keep that in mind only users can see your email (ppl who are logged in github.)

**Comment by ossanna16** [comment]
Thank you for asking a good question @githubbergirl, I'm glad you found our community 🙂
